pants.ini

```diff
@@ -1,6 +1,9 @@
 [GLOBAL]
-pants_version: 1.16.0.dev2
+pants_version: 1.16.0.dev3
 
 plugins: [
     'pantsbuild.pants.contrib.go==%(pants_version)s',
   ]
+
+[python-setup]
+interpreter_constraints: ['CPython>=3.6']
```

Comment:
Would recommend setting a max here, to minimize the chance of being surprised in the future. 4?